src/segment.go

```diff
@@ -112,15 +112,45 @@ func (segment *Segment) getValue(property Property, defaultValue string) string
 	return defaultValue
 }
 
-func (segment *Segment) shouldIgnoreFolder(cwd string) bool {
-	if value, ok := segment.Properties[IgnoreFolders]; ok {
-		list := parseStringArray(value)
-		for _, element := range list {
-			pattern := fmt.Sprintf("^%s$", element)
-			matched := matchString(pattern, cwd)
-			return matched
```

Comment:
BTW I fixed a bug here.  `ignore_folders` was a list, but we only ever processed the first element of that list.

For example, in the below configuration, the segment would be hidden in C:\Projects, but visible in C:\temp

```json
"ignore_folders": [
  "C:\\\\Projects.*",
  "C:\\\\temp.*"
]
```